RecoPixelVertexing/PixelTriplets/interface/HitTripletGeneratorFromPairAndLayers.h

```diff
@@ -33,6 +33,17 @@ class HitTripletGeneratorFromPairAndLayers {
                             const edm::Event & ev, const edm::EventSetup& es,
                             const SeedingLayerSetsHits::SeedingLayerSet& pairLayers,
                             const std::vector<SeedingLayerSetsHits::SeedingLayer>& thirdLayers) = 0;
+
+    virtual void hitTriplets(
+	const TrackingRegion& region, 
+	OrderedHitTriplets & result,
+	const edm::EventSetup & es,
+	const HitDoublets & doublets,
+	const RecHitsSortedInPhi ** thirdHitMap,
+	const std::vector<const DetLayer *> & thirdLayerDetLayer,
+	const int nThirdLayers)
+	{};
```

Comment:
Okay.
Then we'll put dummy implementations with an edm::LogWarning in 
PixelTripletLowPtGenerator and PixelTripletNoTipGenerator.